lib/msal-browser/package.json

```diff
@@ -22,7 +22,7 @@
     "oauth"
   ],
   "sideEffects": false,
-  "main": "./dist/index.cjs.js",
+  "main": "./lib/index.js",
```

Comment:
Why the change from `dist` to `lib`?

Reply:
same q. I thought `dist` is preferred as it will be the final export?

Reply:
2 output formats.... common js and es.... put them in 2 separate folders, since that's what rollup appears to require.